cylc/flow/scripts/reload.py

```diff
@@ -60,19 +60,34 @@
 from cylc.flow.option_parsers import (
     WORKFLOW_ID_MULTI_ARG_DOC,
     CylcOptionParser as COP,
+    OptionSettings,
 )
 from cylc.flow.terminal import cli_function
 
 if TYPE_CHECKING:
     from optparse import Values
 
 
+RELOAD_OPTIONS = [
+    OptionSettings(
+        ['-g', '--global'],
+        help='also reload global configuration.',
+        action="store_true",
+        default=False,
+        dest="reload_global",
+        sources={'reload'}
+    ),
+]
+
+
 MUTATION = '''
 mutation (
-  $wFlows: [WorkflowID]!
+  $wFlows: [WorkflowID]!,
+  $reloadGlobal: Boolean,
 ) {
   reload (
     workflows: $wFlows
+    reloadGlobal: $reloadGlobal
```

Comment:
[In relation to above comment]

I think we can avoid inter-version interoperability issues providing we omit the new `reloadGlobal` field when `--global` was not specified.